### PR TITLE
Add 2023-04-28-newsletter

### PR DESCRIPTION
**Pull request by ptheywood**
Add RSE Newsletter for April 2023.

### CODE REVIEW

File: _newsletters/2023-04-28-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
++ [The Research Software Alliance (ReSA)](https://upstream.force11.org/the-research-software-alliance-resa/)
++ [Code4thought: ByteSized RSE: Integrated Development Environments](https://soundcloud.com/code4thought-615691925/en-bytesized-rse-integrated-development-environments)
++ [Code4thought: Funding it all](https://soundcloud.com/code4thought-615691925/en-funding-it-all)
++ [Code4thought: Who are We?](https://soundcloud.com/code4thought-615691925/en-who-are-we)
++ [MathWorks releases MATLAB extension for Visual Studio Code](https://blogs.mathworks.com/matlab/2023/04/26/do-you-use-visual-studio-code-matlab-is-now-there-too/)
++ [Jisc: Association for Computing Machinery agreement "a significant step forward for open access in the UK"](https://www.jisc.ac.uk/news/association-for-computing-machinery-agreement-a-significant-step-forward-for-open-access-in-the-uk-12-apr-2023?utm_campaign=headlines&utm_content=Headlines%2019%2F04%2F2023&utm_medium=email&utm_source=adestra)
++ [Erasmus Uiversity Rotterdam: Dilemma Game App](https://www.eur.nl/en/about-eur/policy-and-regulations/integrity/research-integrity/dilemma-game)
```
Comment (Robadob) [inline _newsletters/2023-04-28-newsletter.md]: typo university*

File: _newsletters/2023-04-28-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
++ [Ladisa, Piergiorgio, et al. "Journey to the Center of Software Supply Chain Attacks." arXiv preprint arXiv:2304.05200 (2023)](https://arxiv.org/abs/2304.05200). [Interactive Taxonomy Website](https://sap.github.io/risk-explorer-for-software-supply-chains/)
```
Comment (Robadob) [inline _newsletters/2023-04-28-newsletter.md]: Multiple authors doesn't get a colon?

Authors first does read a bit weird in this particular case

Comment (ptheywood) [reply _newsletters/2023-04-28-newsletter.md]: I just grabbed a citation string for it from scholar. Will manually do something that reads better.

**Review by Robadob — COMMENTED** [review]
Rest looks fine.

**Review by Robadob — APPROVED** [review]
I assume you want to get it emailed before Tuesday...

**Comment by EdwinB12** [comment]
@Robadob @ptheywood - This PR doesn't seem to be deploying correctly. . Cant figure out why. . 

https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/actions/runs/4832639369/jobs/8611703925

**Comment by EdwinB12** [comment]
Probabaly should have tried rerunning the workflow before putting in that messsage. . 

All working now :)

**Comment by ptheywood** [comment]
> @Robadob @ptheywood - This PR doesn't seem to be deploying correctly. . Cant figure out why. .
> 
> https://github.com/RSE-Sheffield/RSE-Sheffield.github.io/actions/runs/4832639369/jobs/8611703925

First guess is that there was an gh outage on Friday when I triggered the deployment, so prolly just needs re-running 

Edit: 

> Probabaly should have tried rerunning the workflow before putting in that messsage. .
> 
> All working now :)

Sweet.

**Pull request merged**